source/expressions.tex

```diff
@@ -6771,21 +6771,20 @@
 right operand is a bit-field.
 
 \pnum
-In contexts where comma is given a special meaning,
-\begin{example}
-in
-lists of arguments to functions\iref{expr.call} and lists of
-initializers\iref{dcl.init}
-\end{example}
+In contexts where comma is given special meaning,
 the comma operator as
 described in this subclause can appear only in parentheses.
+\begin{note}
+Such contexts include function calls\iref{expr.call},
+declarators\iref{dcl.decl} and initializer lists\iref{dcl.init.list}.
```

Comment:
A declarator is not a context where an expression can appear naturally, so this addition seems not helpful.
Also, an "initializer list" is not the only place where a lits of initializers can appear, e.g. "T(1,2,3)" is another one.
If you want to add another context, it could be a _template-argument-list_.
Also, I think we can make this entire paragraph a note, because the grammar uses _assignment-expression_ for as the constituents of an _expression-list_; same for _template-argument-list_ (via _constant-expression_). So, this paragraph has zero normative content.
Suggestion: "[ Note: In contexts such as function calls (xref), lists of initializers (xref), or template-argument-lists (xref), the comma operator as
described in this subclause can appear only in parentheses. [ Example: ... ] ]"

Reply:
Good idea